app/src/main/java/me/hufman/androidautoidrive/phoneui/SpotifyAuthorizationActivity.kt

```diff
@@ -60,7 +60,9 @@ class SpotifyAuthorizationActivity: Activity() {
 	private var authIntentLatch = CountDownLatch(1)
 	private val scopes = listOf(
 			SpotifyScope.USER_MODIFY_PLAYBACK_STATE.uri,
-			SpotifyScope.USER_LIBRARY_READ.uri)
+			SpotifyScope.USER_LIBRARY_READ.uri,
+			SpotifyScope.PLAYLIST_MODIFY_PUBLIC.uri,
```

Comment:
Maybe the PLAYLIST_MODIFY_PUBLIC permission isn't needed

Reply:
Yeah it isn't, I accidentally left this in from when I had the playlists created as public in my initial implementation